chalicelib/check_setup.json

```diff
@@ -1,438 +1,516 @@
-                    "kwargs": {"primary": true},
-                    "dependencies": []
-                }
-            },
-            "morning_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "status_of_elasticsearch_indices": {
         "title": "Status of elasticsearch indices",
-        "group": "Elasticsearch checks",
-        "conditions": ["put_env"],
+        "group": "Elasticsearch Checks",
+        "conditions": [
+            "put_env"
+        ],
         "schedule": {
-            "thirty_min_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "indexing_records": {
         "title": "Indexing records",
-        "group": "Elasticsearch checks",
-        "conditions": ["put_env"],
+        "group": "Elasticsearch Checks",
+        "conditions": [
+            "put_env"
+        ],
         "schedule": {
-            "thirty_min_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "change_in_item_counts": {
         "title": "Change in item counts",
-        "group": "Metadata checks",
+        "group": "Metadata Checks",
         "schedule": {
-            "morning_checks_2": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
+                    "kwargs": {
+                        "primary": true
+                    }
                 }
             }
-        }
+        },
+        "display": [
+            "<env-name>"
+        ]
     },
     "page_children_routes": {
         "title": "Pages with bad routes",
-        "group": "Audit checks",
+        "group": "Audit Checks",
         "schedule": {
-            "morning_checks": {
-                "all": {
-                    "kwargs": {"primary": true},
-                    "dependencies": []
+            "manual_checks": {
+                "<env-name>": {
```

Comment:
Checks like this, while probably not needed daily, are useful when run automatically to flag unexpected problems as often when this check returns an issue we haven't thought to run it. If we are just trying to run some checks less frequently to lighten the load on the system, I might suggest changing this to a weekly or monthly schedule.